DS/EmployeeNaturalSorting.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+
+//Class to perform natural sorting on collection of class EmployeeInformation
+public class EmployeeNaturalSorting {
+
+	static List<EmployeeInformation> employeeList = new LinkedList<EmployeeInformation>();
+
+	public static void main(String args[]) {
+		Scanner scan = new Scanner(System.in);
+		Menu menu = new Menu();
+		int choice;
+		int ch = 0;
+		do {
+			// Calling display function
+			menu.display();
+			do {// Validation
+				while (!scan.hasNextInt()) {
+					System.out
+							.println("That's not a number!Please enter again");
+					scan.next(); // this is important!
+				}
+				choice = scan.nextInt();
+			} while (choice <= 0);
+			choice = scan.nextInt();
+			switch (choice) {
+			case 1:
+				int count = 0;
+				EmployeeInformation obj = menu.inputEmployeeInformation();
```

Comment:
ACTION: Pass "scan" as parameter in method menu.inputEmployeeInformation() to take input from user instead create new object of scanner in menu class.